crates/forge/tests/e2e/running.rs

```diff
@@ -150,7 +150,7 @@ fn with_failing_scarb_build() {
         output,
         indoc!(
             r"
-                [ERROR] Failed to build test artifacts with Scarb: `scarb` exited with error
+                [ERROR] Failed to build contracts with Scarb: `scarb` exited with error
```

Comment:
Why this error changed? Is it just because order of compilation changed?

Reply:
Yup exactly, the order of compilation changed. Once we update scarb to 2.8.3 in the repository the error will be for "test artifacts" again